flocker/control/_model.py

```diff
@@ -199,6 +201,7 @@ class Application(PRecord):
     restart_policy = field(mandatory=True, initial=RestartNever())
     environment = field(mandatory=True, initial=pmap(), factory=pmap,
                         type=PMap)
+    running = field(mandatory=True, initial=True, type=bool)
 
```

Comment:
> but also a bunch of code elsewhere e.g. the create a container endpoint.

Yea... that's the kind of thing that worries me.  Whether you get a running container or a not running container by using the API doesn't seem like it should depend on the default value we put into a model type.